src/main/resources/staff/templates/email/ccjRequestSubmitted/body.txt

```diff
@@ -1,5 +1,5 @@
 The claimant has asked for a County Court Judgment to be made against the defendant.
 
-The claimant, who is making this claim {{ claimantType }}, has requested that the debt be paid {{ paymentType }}.
+The attached PDF explains how the claimant wants to be paid and includes the defendant’s address.
```

Comment:
Where is the reference for this copy change?

Reply:
https://www.notifications.service.gov.uk/services/37ba0bfe-68ed-4e8e-9e19-f3a4cc331d80/templates/906491d7-f118-4381-96d2-e890d99ac1e8